lib/browser/guest-view-manager.ts

```diff
@@ -155,8 +155,12 @@ const createGuest = function (embedder: Electron.WebContents, embedderFrameId: n
   });
 
   // Dispatch guest's IPC messages to embedder.
-  guest.on('ipc-message-host' as any, function (_: Electron.Event, channel: string, args: any[]) {
-    sendToEmbedder(IPC_MESSAGES.GUEST_VIEW_INTERNAL_IPC_MESSAGE, channel, ...args);
+  guest.on('ipc-message-host' as any, function (event: Electron.IpcMainEvent, channel: string, args: any[]) {
+    sendToEmbedder(IPC_MESSAGES.GUEST_VIEW_INTERNAL_IPC_MESSAGE, {
+      frameId: [event.processId, event.frameId],
```

Comment:
I think I'd be happier  with something like this so we won't need to introduce another API method.
```suggestion
      get frame() {
        return webFrameMain.fromId(event.processId, event.frameId)
      },
```

Reply:
Ah, I understand now that this API is being introduced to the renderer. :)

Reply:
we are emitting the `<webview>` event in the renderer process, there is no `webFrameMain` there